### ISSUE TITLE
Crash inside docker container

### ISSUE THREAD

**Issue by jsmouret** [Alexpux/MSYS2-packages#1239]
Crash reports:
```

**Comment by esman** [comment]
@jsmouret Hi!
Could you try:
1. Run the container interactively
Copy the whole msys folder. e.g. `xcopy c:\msys64 c:\msys64_copy`
Run command from copied folder: `c:\msys64_copy\usr\bin\ls.exe`

**Comment by jsmouret** [comment]
Minus the fact that stdout doesn't show, it seems to work !

```
C:\Windows\Temp>c:\msys64_copy\usr\bin\ls.exe > out

C:\Windows\Temp>type out
MpCmdRun.log
msys2.tar
msys2.tar.xy
out
silconfig.log
```

Any idea why?

**Comment by esman** [comment]
Not really. It looks like msys or some it's dependency are using comprehensive dll mechanism which is broken for commited fs part.
The upper hack works not only when copying during interactive session, but also if to use directory mounts with '-v' option.

**Comment by amirhadar** [comment]
I have the same issue with cygwin (or git\usr\ls) within Windows Docker 2016.
Any suggestions?

**Comment by cmeiklejohn** [comment]
I'm also experiencing this trying to run iperf3.exe inside of the container -- same exit code.

**Comment by Cherishty** [comment]
@esman  
It works! And mkdir.exe also works after copying folder. But bash.exe still not work if I try `usr/bin/bash.exe -l -c 'echo 0'` or `usr/bin/bash.exe -l -c 'mkdir test'` .
BTW , this is what I want , run GNU bash command in windows container, any suggestion?

**Comment by tatototino** [comment]
> I'm also experiencing this trying to run iperf3.exe inside of the container -- same exit code.

I have the same issue, I can´t make iperf3 running in nanoserver container.

**Comment by filipesilva** [comment]
I've added some more detail to stdout not showing in https://github.com/Alexpux/MSYS2-packages/issues/1490.
